i18n/en.json

```diff
@@ -1293,9 +1293,11 @@
   "oldest_first": "Oldest first",
   "on_this_device": "On this device",
   "onboarding": "Onboarding",
-  "onboarding_privacy_description": "The following (optional) features rely on external services, and can be disabled at any time in the administration settings.",
+  "onboarding_locale_description": "Select your preferred language. You can change this later in your settings.",
+  "onboarding_privacy_description": "The following (optional) features rely on external services, and can be disabled at any time in settings.",
```

Comment:
Keeping the same key while changing the actual value isn't ideal, since this kind of changes won't get picked up by translators.